lib/test-utils.js

```diff
@@ -0,0 +1,36 @@
+var test = require('tape');
+var path = require('path');
+var defined = require('defined');
+var Levenshtein = require('levenshtein');
+
+test.Test.prototype.equalPaths = function (a, b, msg, extra) {
```

Comment:
let's please not mutate objects we don't own :-) i'd prefer a pure function that takes a `t` along with all the relative arguments.